cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -228,6 +229,15 @@ public void execute() {
             }
         }
 
+        if (isProjectEmpty(project)) {
+            if (!(this.compile && project.currentPackage().compilerPluginToml().isPresent())) {
```

Comment:
Let's switch these conditions so that we can avoid iterating the modules every time.

Reply:
Switched the conditions as requested